app/api/drink_routes.py

```diff
@@ -0,0 +1,67 @@
+from flask import Blueprint, request, jsonify
+from app.models import db, Menu, Business, Toppings, Drinks
+from ..forms.drink_form import DrinkForm
+from flask_login import current_user, login_required
+from .auth_routes import validation_errors_to_error_messages
+
+drink_routes = Blueprint("drinks", __name__, url_prefix="/drinks")
+
+# Get all drinks + toppings, needs to be tested
+@drink_routes.route("")
+def all_drinks_toppings():
+    drinks = Drinks.query.all()
+    toppings = Toppings.query.all()
+    return {
+        "drinks": [d.to_dict() for d in drinks],
+        "toppings": [t.to_dict() for t in toppings]
+            }
+
+# Create drink
+@drink_routes.route("/", methods=["POST"])
+@login_required
+def create_drink():
+    form = DrinkForm()
+    form["csrf_token"].data = request.cookies["csrf_token"]
```

Comment:
Validate user to be owner of the business / menu 

Only owner can add drink to a menu